### PR TITLE
Gives posibrains a time requirement just like the cyborg job

### PR DESCRIPTION
**Pull request by Moltijoe**
image

:cl:  
rscadd: Time requirement to posibrains
/:cl:

### CODE REVIEW

**Comment by MajManatee** [comment]
Possibly out of scope, but would it be good to have a seperate timer for posi brains specifically, and the ability to ban from the job?

File: code/modules/mob/living/brain/posibrain.dm [hunk 1]
```diff
@@ -97,6 +97,10 @@ GLOBAL_VAR(posibrain_notify_cooldown)
 	if(user.suiciding) //if they suicided, they're out forever.
 		to_chat(user, span_warning("[src] fizzles slightly. Sadly it doesn't take those who suicided!"))
 		return
+	if(user.client.get_exp_living(TRUE) < SSjob.GetJob("Cyborg").exp_requirements && !(CONFIG_GET(flag/use_exp_restrictions_admin_bypass) && check_rights_for(user.client, R_ADMIN)))
```
Comment (ThatLing) [inline code/modules/mob/living/brain/posibrain.dm]: Use ``/datum/job/proc/required_playtime_remaining``

File: code/modules/mob/living/brain/posibrain.dm [hunk 2]
```diff
@@ -97,6 +97,10 @@ GLOBAL_VAR(posibrain_notify_cooldown)
...
+		to_chat(user, span_warning("Positron brains are beyond your knowledge to control."))
```
Comment (ThatLing) [inline code/modules/mob/living/brain/posibrain.dm]:
```suggestion
		to_chat(user, span_warning("Positronic brains are beyond your knowledge to control."))
```

File: code/modules/mob/living/brain/posibrain.dm [hunk 3]
```diff
@@ -97,6 +97,10 @@ GLOBAL_VAR(posibrain_notify_cooldown)
...
+		to_chat(user, span_warning("In order to play as a positron brain, you require [SSjob.GetJob("Cyborg").exp_requirements - user.client.get_exp_living(TRUE)] more minutes of experience on-board the station."))
```
Comment (ThatLing) [inline code/modules/mob/living/brain/posibrain.dm]: Use returned value from ``/datum/job/proc/required_playtime_remaining``

**Review by ThatLing — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/brain/posibrain.dm [hunk 4]
```diff
@@ -97,6 +97,10 @@ GLOBAL_VAR(posibrain_notify_cooldown)
 	if(user.suiciding) //if they suicided, they're out forever.
 		to_chat(user, span_warning("[src] fizzles slightly. Sadly it doesn't take those who suicided!"))
 		return
+	if(SSjob.GetJob("Cyborg").required_playtime_remaining(user.client) && !(CONFIG_GET(flag/use_exp_restrictions_admin_bypass) && check_rights_for(user.client, R_ADMIN)))
```
Comment (ThatLing) [inline code/modules/mob/living/brain/posibrain.dm]: The additional checks are already performed in ``required_playtime_remaining``

File: code/modules/mob/living/brain/posibrain.dm [hunk 5]
```diff
@@ -97,6 +97,10 @@ GLOBAL_VAR(posibrain_notify_cooldown)
...
+		to_chat(user, span_warning("Positronic brains are beyond your knowledge to control."))
+		to_chat(user, span_warning("In order to play as a positron brain, you require [SSjob.GetJob("Cyborg").required_playtime_remaining(user.client)] more minutes of experience on-board the station."))
```
Comment (ThatLing) [inline code/modules/mob/living/brain/posibrain.dm]: Cache the return value

**Review by ThatLing — CHANGES_REQUESTED** [review]

**Review by ThatLing — APPROVED** [review]

**Review by ThatLing — APPROVED** [review]

**Review by ThatLing — APPROVED** [review]

**Pull request merged**